### ISSUE TITLE
pathlib glob ignores trailing slash in pattern

### ISSUE THREAD

**Issue by 451a3d21-cfd7-40f5-aaa4-d20c47ed1fa4** [python/cpython#66472]
BPO | [22276](https://bugs.python.org/issue22276)
--- | :---
Nosy | @pitrou, @bitdancer, @emilyemorehouse, @robbuckley, @akulakov
PRs | <li>python/cpython#10349</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/emilyemorehouse'
closed_at = None
created_at = <Date 2014-08-26.11:37:22.662>
labels = ['3.8', 'type-bug', 'library']
title = 'pathlib glob ignores trailing slash in pattern'
updated_at = <Date 2021-11-21.02:47:05.959>
user = 'https://bugs.python.org/jocabt'
```

bugs.python.org fields:
```python
activity = <Date 2021-11-21.02:47:05.959>
actor = 'andrei.avk'
assignee = 'emilyemorehouse'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2014-08-26.11:37:22.662>
creator = 'joca.bt'
dependencies = []
files = []
hgrepos = []
issue_num = 22276
keywords = ['patch']
message_count = 9.0
messages = ['225914', '226033', '226034', '226036', '226095', '226102', '406700', '406701', '406702']
nosy_count = 7.0
nosy_names = ['pitrou', 'r.david.murray', 'cvrebert', 'joca.bt', 'emilyemorehouse', 'robbuckley', 'andrei.avk']
pr_nums = ['10349']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue22276'
versions = ['Python 3.8']
```

</p></details>

**Comment by barneygale** [comment]
This was fixed in #10349. This issue could be closed.

**Comment by barneygale** [comment]
Fixed in #10349.
